### ISSUE TITLE
Focus goes from "Vary by configuration" menu in project property page directly to Solution Explorer "Home" button after selecting an option via keyboard

### ISSUE THREAD

**Issue by japarson** [dotnet/project-system#8002]
## Visual Studio Version
Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview
Version 17.2.0 Preview 2.0

## Summary
Focus goes from "Vary by configuration" menu in project property page directly to Solution Explorer "Home" button after selecting an option via keyboard.
- [Bug 1504052](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1504052): A11y_visual studio client_IOS IPA options_build IPA_Focus order: After selecting option from menu item, focus goes to "home" button instead of staying on the "menu item" .

https://user-images.githubusercontent.com/59936622/159622921-5a9f85fb-8ca3-4261-a501-479b70860c61.mp4

## Steps to Reproduce
1. Create new iOS project from power shell using `dotnet new android -n <project_name>`.
1. Open VS 2022 and tab till "Open a project or solution" option and press enter.
1. Tab till "File name" and select newly created iOS "csproj" file and press enter on "Open" button.
1. Tab till project in "Solution explorer" and press context key.
1. Tab till "Properties" menu item and press enter.
1. Tab till "iOS IPA options" button and press enter.
1. Tab till "Build IPA" present in "IPA packaging options" section.
1. Tab till "menu item" present beside the "Build IPA" label and press enter.
1. Now, select an option from the  menu item and press tab.
1. Verify whether the focus stays on the menu item or not.

## Expected Behavior
1. After selecting the option from the "menu item" dropdown, focus should stay on the "menu item" and it should not go to "Home" button.
2. Focus should be in order.

## Actual Behavior
After selecting option from menu item, focus goes to "Home" button instead of staying on the "menu item".

## User Impact
After selecting the options from that menu item, keyboard users will face challenges while navigating to the next controls, which are present after "Build IPA" menu item. If the focus goes to the "home" button, after selecting the items from the "Build IPA" menu item then user need to again visit all the previous controls, if the user wants to move the "focus" on the further controls.

[MAS 2.4.3 – Focus Order](https://aka.ms/MAS2.4.3)
